src/components/MultiHttp/Tabs/AssertionsTab.tsx

```diff
@@ -143,82 +167,94 @@ function AssertionFields(props: AssertionProps) {
 
 function AssertionSubjectField({ entryIndex, assertionIndex }: AssertionProps) {
   const { formState } = useFormContext<CheckFormValuesMultiHttp>();
-  const error = formState.errors.settings?.multihttp?.entries?.[entryIndex]?.checks?.[assertionIndex]?.subject;
-  const errMessage = error?.message;
+  const error = formState.errors.settings?.multihttp?.entries?.[entryIndex]?.checks?.[assertionIndex];
+  const err = errorHasSubject(error) ? error?.subject : undefined;
```

Comment:
Can we rename `err` to `errorSubject` or similar? I find it a bit confusing to see both `error` and `err`